data/Course Details/TempEdit.txt

```diff
@@ -0,0 +1,3 @@
+Thursday 8:00am-10:00am
```

Comment:
Why is it called `TempEdit`.txt

Reply:
I actually restructured my code and this file is no longer needed anymore but I forgot to delete this txt file

Reply:
Please delete and push again